### PR TITLE
Part 3

### PR DESCRIPTION
**Pull request by brsoyann**

### CODE REVIEW

File: ToDo/Architecture/Controllers/ViewControllers/BooksToReadTableViewController.swift [hunk 1]
```diff
@@ -15,7 +15,10 @@ final class BooksToReadTableViewController: UITableViewController {
```
Comment (Aslanyan99n) [inline ToDo/Architecture/Controllers/ViewControllers/BooksToReadTableViewController.swift:15]: avel funcia?

Comment (Aslanyan99n) [inline ToDo/Architecture/Controllers/ViewControllers/BooksToReadTableViewController.swift]: datark func unwindToBookListSegue jnji

Comment (Aslanyan99n) [reply ToDo/Architecture/Controllers/ViewControllers/BooksToReadTableViewController.swift]: layout

es layoutd karas dzes checkmarky vor phoqr tex zbaxecni mnacacy texy ta textfieldnerin

Comment (Aslanyan99n) [reply ToDo/Architecture/Controllers/ViewControllers/BooksToReadTableViewController.swift]: Screen Shot 2022-10-05 at 23 32 05

stex checkmard chi ashxatum

Comment (brsoyann) [reply ToDo/Architecture/Controllers/ViewControllers/BooksToReadTableViewController.swift]: unwindToBookListSegue- ը cancel ի համար ա

**Review by Aslanyan99n — APPROVED** [review]

**Comment by Aslanyan99n** [comment]
ok

**Comment by Aslanyan99n** [comment]
vsyo merge ara

**Pull request merged**